### ISSUE TITLE
Remove org.kde.* permission

### ISSUE THREAD

**Issue by bilelmoussaoui** [flathub/com.dropbox.Client#111]
The app shouldn’t have the possibility to own any dbus name under org.kde.*

**Comment by wjt** [comment]
Like many Qt apps which want to show indicator icons, this app attempts to own a name of the form `org.kde.StatusNotifierItem-$PID-1`. With https://github.com/ubuntu/gnome-shell-extension-appindicator installed, this allows Dropbox to show a tray icon. It falls back to xembed, for which one also needs an extension on GNOME. If neither works, the app shows no UI whatsoever and is unusable.

**Comment by wjt** [comment]
One option would be to allow it to own a whole bunch of names of the form `org.kde.StatusNotifierItem-$PID-1` and manually update the list in response to bug reports. (When this was added, the PID that ended up assigned to Dropbox was different for the same Flatpak on different distros (!).)

**Comment by bilelmoussaoui** [comment]
Indeed, the situation is a bit messy around tray icons. I have opened the issue just to keep track of it.
